Guided Investigation - Azure WAF SQLI.ipynb

```diff
@@ -0,0 +1,792 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.            qry = f"SecurityAlert | where TimeGenerated between((datetime({start})-7d)..datetime({end})) | where SystemAlertId == '{alert['ID']}'"
</pre><p>Does nested 'ID' string work? in "where SystemAlertId == '{alert['ID']}'" - seems like it shouldn't</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
It does work thankfully!